nefertari/authentication/__init__.py

```diff
@@ -0,0 +1,24 @@
+def includeme(config):
+    """ Set up event subscribers. """
+    from .models import (
+        AuthUserMixin,
+        random_uuid,
+        lower_strip,
+        encrypt_password,
+    )
+    from nefertari import events
+
+    subscribe_to = (
+        events.before_create,
+        events.before_update,
+        events.before_replace,
+        events.before_update_many,
+    )
+
+    add_sub = config.add_subscriber
```

Comment:
I wonder if we want a re-usable function for registering field processors. They are kind of different than regular event subscribers in that you always want to register them for the same group of events. Also this may make it easier to do this kind of thing in ramses.